lib/linguist/languages.yml

```diff
@@ -5881,6 +5883,8 @@ XML:
   - wsdl
   extensions:
   - ".xml"
+  - ".4DCatalog"
+  - ".4DSettings"
```

Comment:
All these extensions except `.4DForm` have less than 200 files, so they clearly don't meet our requirements of *hundreds of repositories*. Could you please remove the last three? Do you expect usage to grow significantly in the next few months for any of them?

Reply:
There is only one 4dcatalog and one 4dsettings for a project. So yes the number of files are less than 200. But we expect a lot of submit in the next months.

Reply:
Because the project model is an evolution of a binary package. A great evolution of the 4D development environment.

Reply:
You can find a sample here : https://github.com/vdelachaux/4DPop-Git/Project/Sources/Forms/GIT/form.4DForm

Reply:
But I can remove this entries if you think it’s better